Automattic/wp-calypso
# Hosting Metrics: Add two new multi line charts

Related to #https://github.com/Automattic/dotcom-forge/issues/3274

## Proposed Changes

This PR:

Check out pdKhl6-2cB-p2#comment-3799

* Add support for multiple lines in a single chart.
* Adds two new  line charts, success and error responses to hosting metrics
* Removes the bar chart

<img width="1280" alt="Screenshot 2023-08-09 at 2 53 09 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/c25f15ab-e474-4dc1-a124-36d19d02a3bf">

* adds two new line charts
<img width="1707" alt="Screenshot 2023-08-11 at 16 55 44" src="https://github.com/Automattic/wp-calypso/assets/779993/83f4875f-6f7a-407b-8d33-6fefac90fb14">


## Testing Instructions

* Navigate to Calypso Live Link
* Append `site-monitoring/{yourAtomicSite}page=metrics`
* Observe two lines on the `Requests per minute and average response time` -the blue one for requests per minute and the green one for average response time
* Scroll to the bottom of the page
* Observe the new two line charts with multiple lines
* Try changing options on the time/date picker and confirm that the charts change


## Screencast

https://github.com/Automattic/wp-calypso/assets/779993/412ebe4d-6d55-4b8a-93c9-61a8cfd58acf


**Known Issues**:

* The design of the legend is not yet consistent with the proposed design
* The charts are now taking the UTC timezone by default instead of the site's timezone
* The formatting of the Y axis needs to be adjusted to `ms` on the first chart

## Pre-merge Checklist
